### PR TITLE
Added Okta Verify with MFA post

### PR DESCRIPTION
**Pull request by dogeared**
## Description:
- Post with link to screencast on using Okta Verify with API

**NOTE: scheduled for release on 2/7/18**

### CODE REVIEW

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+---
+layout: blog_post
+title: 'Set Up and Enforce Multi-Factor Authentication with the Okta API'
+author: dogeared
+tags: [security, mfa, multi-factor authentication, authentication, api]
+tweets:
+  - "Love APIs? Watch as @afitnerd walks you through MFA setup with the @Okta API."
```
Comment (nbarbettini) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: Suggestion: "Love APIs and MFA?"

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  - "@OktaDev <3 APIs. Watch as @afitnerd walks you through MFA setup with the @Okta API."
+---
+
+So, you're building a custom app and you need to include support for multi-factor authentication (MFA). Did you know that [Okta's API](https://developer.okta.com/) can support multi-step MFA workflows?
+
+The diagram below shows the primary authentication flows with MFA enrollment and enforcement enabled. 
+
+{% img blog/use-okta-api-for-mfa/mfa_enroll_enforce.png alt:"With API" width:"700" %}{: .center-image }
+
+Okta has a time-based one-time password soft token app called Okta Verify. It's available on the [Google Play](https://play.google.com/store/apps/details?id=com.okta.android.auth&hl=en) and [Apple App](https://itunes.apple.com/us/app/okta-verify/id490179405?mt=8) stores. Okta Verify is similar to Google Authenticator, except it adds the additional feature of supporting push notifications. We think this strikes a really great balance between additional security and user experience.
```
Comment (nbarbettini) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: "push-based verification app" might be clearer than "time-based on-time password soft token app"

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+I created a screencast to demonstrate how you can manage the whole MFA enrollment and enforcement process using the Okta API:
+
+<div style="width: 560px; margin: 0 auto">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/EVL3gnt7BYo" frameborder="0" allowfullscreen></iframe>
+</div>
+
+Interested in learning more about secure authentication with Okta? Check out our [Product Documentation](https://developer.okta.com/documentation/), our hands-on [Platform for Developers](https://www.okta.com/services/training/) course, or any of these great posts from our developer blog:
+* [Build an App for iOS and Android with Xamarin](https://developer.okta.com/blog/2018/01/10/build-app-for-ios-android-with-xamarin)
+* [Get Started with Spring Security 5.0 and OIDC](https://developer.okta.com/blog/2017/12/18/spring-security-5-oidc)
+* Use Kong Gateway to Centralize Authentication](https://developer.okta.com/blog/2017/12/04/use-kong-gateway-to-centralize-authentication)
```
Comment (nbarbettini) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: nit: missing opening `[`

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+* Add Auth to Play Framework with OIDC and Okta](https://developer.okta.com/blog/2017/10/31/add-authentication-to-play-framework-with-oidc)
```
Comment (nbarbettini) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: nit: missing opening `[`

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+* Use OpenID Connect to Build a Simple Node.js Website](https://developer.okta.com/blog/2017/10/19/use-openid-connect-to-build-a-simple-node-website)
```
Comment (nbarbettini) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: nit: missing opening `[`

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
+So, you're building a custom app and you need to include support for multi-factor authentication (MFA). Did you know that [Okta's API](https://developer.okta.com/) can support multi-step MFA workflows?
+
+The diagram below shows the primary authentication flows with MFA enrollment and enforcement enabled. 
+
+{% img blog/use-okta-api-for-mfa/mfa_enroll_enforce.png alt:"With API" width:"700" %}{: .center-image }
+
+Okta has a time-based one-time password soft token app called Okta Verify. It's available on the [Google Play](https://play.google.com/store/apps/details?id=com.okta.android.auth&hl=en) and [Apple App](https://itunes.apple.com/us/app/okta-verify/id490179405?mt=8) stores. Okta Verify is similar to Google Authenticator, except it adds the additional feature of supporting push notifications. We think this strikes a really great balance between additional security and user experience.
+
+I created a screencast to demonstrate how you can manage the whole MFA enrollment and enforcement process using the Okta API:
```
Comment (mraible) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: I'd give a link to the YouTube video and say "or you can watch it below".

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+
+<div style="width: 560px; margin: 0 auto">
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/EVL3gnt7BYo" frameborder="0" allowfullscreen></iframe>
```
Comment (mraible) [inline _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: You could make this bigger if you like. I think our default width is pretty wide.

File: _source/_posts/2018-02-07-use-okta-api-for-mfa.md (hunk 4 above)
Comment (mraible) [reply _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: Blog post links should be relative, starting with /blog

Comment (mraible) [reply _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: For that matter, you don't need absolute links for anything under developer.okta.com

Comment (robertjd) [reply _source/_posts/2018-02-07-use-okta-api-for-mfa.md]: Yep, using relative makes all the things play nice in CI & staging

**Comment by dogeared** [comment]
just saw the comments. I'll updated before the 7th. Thanks.

**Comment by dogeared** [comment]
updated per comments from @mraible and @natebarbettini-okta. @mraible - would you please push live on 2/7? Thanks.

**Pull request merged**